doc/profiling.md

```diff
@@ -0,0 +1,54 @@
+Profiling
+=========
+
+There's two ways to profile the game to figure out what is causing
```

Comment:
I think "There are two ways" sounds better here.

Reply:
I think it's good enough. I've written most of the other documentation in my personal writing style anyway...